src/node_mgmt/env_scan.py

```diff
@@ -360,6 +361,11 @@ def __init__(self, ifname=None, serial_dev=None):
 
         self.net_env = NetworkEnv(default_ifname=ifname)
         self.serial_env = SerialEnv(default_serial_dev=serial_dev)
+        self.speedwire_env = SpeedWireReader()
+
+    def speedwire_scan(self):
```

Comment:
[1/2] I'm not sure we need a dedicated `self.speedwire_scan()` method, since what it does is kind of trivial...